packages/backend-defaults/src/entrypoints/urlReader/lib/AzureBlobStorageUrlReader.ts

```diff
@@ -171,7 +171,7 @@ export class AzureBlobStorageUrlReader implements UrlReaderService {
         },
       );
     } catch (e) {
-      if (e.$metadata && e.$metadata.httpStatusCode === 304) {
+      if (e.statusCode === 304) {
```

Comment:
Could you please provide a documentation link for the expected status codes? I was just wondering how you find that this is the right way to detect a modified response error :) I haven't used this client before, so I'm just curious.